napari/layers/image/image.py

```diff
@@ -225,16 +226,15 @@ def __init__(
         self._data_thumbnail = self._data_view
 
         # Set contrast_limits and colormaps
-        self._gamma = gamma
-        self._iso_threshold = iso_threshold
-        self._attenuation = attenuation
+        self._on_gamma(gamma)
+        self._on_iso_threshold_change(iso_threshold)
+        self._on_attenuation_change(attenuation)
```

Comment:
Some update functions are called `on_attr_change` while others are `on_attr`... Should we unify these?